docs/src/_reference/command-line-interface.md

```diff
@@ -732,11 +731,69 @@ meltano job remove <job_name>
 
 ### Examples
 
+##### Tasks and Blocks
+
+A task in a job consists of one or more _blocks_. A block is simply a set of related plugins that will be executed serially.
```

Comment:
They're plugin _commands_, not just plugins, e.g. `dbt:run`. Can we specifically refer to the `meltano run` syntax/docs since this (I think) is modeled after that?

Reply:
Also, I don't think "related" here helps, they're not just "related plugins", they're steps in a pipeline.